extensions/wikia/PhalanxII/hooks/PhalanxHooks.class.php

```diff
@@ -135,7 +135,7 @@ static public function onEditPhalanxBlock( &$data ) {
 
 		if ( $phalanx['expire'] === '' || is_null( $phalanx['expire'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expire" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)